docs/tutorials/blog.md

```diff
@@ -882,13 +886,17 @@ export default function NewPost() {
 
 TypeScript is still mad, so let's add some invariants to make it happy.
 
-```tsx filename=app/routes/admin/new.tsx lines=[11-14,16-18]
```

Comment:
I'm not 100% on these line numbers. I didn't actually run the build and run the docs to observe. If someone can verify these that would be nice!